microsoft/testsuites/core/timesync.py

```diff
@@ -313,8 +313,11 @@ def verify_timesync_unbind_clockevent(self, node: Node) -> None:
         priority=2,
     )
     def verify_timesync_ntp(self, node: Node) -> None:
-        if isinstance(node.os, Redhat) and node.os.information.version >= "8.0.0":
-            # refer from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/using-chrony-to-configure-ntp # noqa: E501
+        # refer from https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/using-chrony-to-configure-ntp # noqa: E501
+        # refer https://documentation.suse.com/sles/15-SP4/html/SLES-all/cha-ntp.html#sec-ntp-chrony # noqa: E501
```

Comment:
The document is too long to read. Please summarize the reason to skip the test case on higher version OSes.

Reply:
updated